Snabble/UI/Scanner/CheckoutBar.swift

```diff
@@ -29,44 +30,147 @@ final class CheckoutBar: NibView {
-
-        self.methodSelectionStackView.layer.masksToBounds = true
-        self.methodSelectionStackView.layer.cornerRadius = 8
-        self.methodSelectionStackView.layer.borderColor = UIColor.lightGray.cgColor
-        self.methodSelectionStackView.layer.borderWidth = 1 / UIScreen.main.scale
-
-        self.noPaymentLabel.text = L10n.Snabble.Shoppingcart.BuyProducts.selectPaymentMethod
-
-        self.totalPriceLabel.font = UIFont.monospacedDigitSystemFont(ofSize: 22, weight: .bold)
-
-        self.methodSelector = PaymentMethodSelector(parentVC, self.methodSelectionStackView!, self.methodIcon, self.shoppingCart)
-        self.methodSelector?.delegate = self
+        checkoutButton.setTitleColor(disabledColor, for: .disabled)
+        checkoutButton.titleLabel?.font = UIFont.systemFont(ofSize: 17, weight: .semibold)
+        checkoutButton.makeSnabbleButton()
+        checkoutButton.isEnabled = true
+        checkoutButton.isUserInteractionEnabled = true
+        checkoutButton.addTarget(self, action: #selector(checkoutTapped(_:)), for: .touchUpInside)
+
+        addSubview(contentStack)
+
+        contentStack.addArrangedSubview(summaryView)
+        contentStack.addArrangedSubview(paymentStackView)
+        summaryView.addSubview(itemCountLabel)
+        summaryView.addSubview(totalPriceLabel)
+        paymentStackView.addArrangedSubview(methodSelectionStackView)
+        paymentStackView.addArrangedSubview(checkoutButton)
+        methodSelectionStackView.addArrangedSubview(noPaymentLabel)
+        methodSelectionStackView.addArrangedSubview(methodIcon)
+        methodSelectionStackView.addArrangedSubview(shevronImage)
+
+        self.contentStack = contentStack
+        self.summaryView = summaryView
+        self.itemCountLabel = itemCountLabel
+        self.totalPriceLabel = totalPriceLabel
+        self.paymentStackView = paymentStackView
+        self.methodSelectionStackView = methodSelectionStackView
+        self.noPaymentLabel = noPaymentLabel
+        self.methodIcon = methodIcon
+        self.shevronImage = shevronImage
+        self.checkoutButton = checkoutButton
+
+        NSLayoutConstraint.activate([
+            contentStack.topAnchor.constraint(equalTo: topAnchor),
+            contentStack.bottomAnchor.constraint(equalTo: bottomAnchor),
+            contentStack.leadingAnchor.constraint(equalTo: leadingAnchor),
+            contentStack.trailingAnchor.constraint(equalTo: trailingAnchor),
+
+            summaryView.heightAnchor.constraint(equalToConstant: 26).usingPriority(.init(rawValue: 999)),
+            itemCountLabel.leadingAnchor.constraint(equalTo: summaryView.leadingAnchor),
+            itemCountLabel.centerYAnchor.constraint(equalTo: summaryView.centerYAnchor),
+            totalPriceLabel.trailingAnchor.constraint(equalTo: summaryView.trailingAnchor),
+            totalPriceLabel.centerYAnchor.constraint(equalTo: summaryView.centerYAnchor),
```

Comment:
We want to support dynamic fonts in the future. To make it easier in the future could you please attach all elements to all four edges. If we add dynamic font it will break at the moment, cause `itemCountLabel` would increase too much.